link/perf_event.go

```diff
@@ -204,6 +217,25 @@ func getPMUEventType(pmu string) (uint64, error) {
 	return et, nil
 }
 
+// determineRetprobeBit reads a Performance Monitoring Unit's retprobe (numeric identifier)
+// from /sys/bus/event_source/devices/<pmu>/format/retprobe.
+func determineRetprobeBit(pmu string) (uint64, error) {
```

Comment:
Could this be done in a separate PR?

Reply:
https://github.com/cilium/ebpf/pull/282

Reply:
This function should also wrap a `sync.Once` since the value is static. (see `var kernelVersion` for a simple example)

Reply:
Resolving, let's continue in the dedicate PR